tests/modules/sratools/prefetch/nextflow.config

```diff
@@ -1,3 +1,6 @@
+params.settings_path = '/tmp/.ncbi'
+params.settings_file = "${params.settings_path}/user-settings.mkfg"
```

Comment:
This is getting increasingly complex to follow 😏 Would it be easier to provide `ncbi_settings` as input and if it is empty e.g. `[]` then to create one. If it isn't then to use that?

Reply:
To me the other configuration is the more complex one. I'm testing two branches of logic,

1. The settings are supplied as a file.
2. The settings are available on the filesystem (or mounted into the container), in a location that can be discovered by the sra-tools.

The test cases use these parameters. So to my mind it makes more sense to set the path in a single location rather than hard code it everywhere.

Reply:
We can add the file directly as a test-dataset and update [this](https://github.com/nf-core/modules/blob/master/tests/config/test_data.config) file so it truly is set and available from a single location (even for multiple modules)?

Reply:
Oh yeah, good call. Still need to download it to a sensible location on the local filesystem, though, for testing one branch of the logic. So I don't think I can entirely avoid setting parameters.